docs/Users/Tutorials/External_Bears.rst

```diff
@@ -0,0 +1,511 @@
+External Bears
+==============
+
+Welcome. This tutorial hopes to teach you how to use the
```

Comment:
cmon you cant use in a tutorial ``hopes to teach you`` :p

try: This tutorial is made to teach you..